src/components/views/elements/ReplyThread.tsx

```diff
@@ -354,13 +358,22 @@ export default class ReplyThread extends React.Component<IProps, IState> {
         }
 
         const evTiles = this.state.events.map((ev) => {
-            return <blockquote className={`mx_ReplyThread ${this.getReplyThreadColorClass(ev)}`} key={ev.getId()}>
-                <ReplyTile
-                    mxEvent={ev}
-                    onHeightChanged={this.props.onHeightChanged}
-                    permalinkCreator={this.props.permalinkCreator}
-                />
-            </blockquote>;
+            const classname = classNames({
+                'mx_ReplyThread': true,
+                [this.getReplyThreadColorClass(ev)]: true,
+                'mx_ReplyThread--expanded': this.props.isThreadExpanded === true,
+                'mx_ReplyThread--collapsed': this.props.isThreadExpanded === false,
```

Comment:
```suggestion
                'mx_ReplyThread--expanded': this.props.isThreadExpanded,
                'mx_ReplyThread--collapsed': !this.props.isThreadExpanded,
```

Reply:
We don't want to show either of those if it can be undefined, so either this or explicit `this.props.isThreadExpanded !== undefined`

Reply:
Ah, sorry makes sense :man_facepalming: